Open Judge System/Data/OJS.Data.Models/Problem.cs

```diff
@@ -62,6 +62,8 @@ public class Problem : DeletableEntity, IOrderable
         [DefaultValue(false)]
         public bool ShowDetailedFeedback { get; set; }
 
+        public string DefaultSubmissionTypeName { get; set; }
```

Comment:
This better be Id. What will happen if the submission type is renamed?

Reply:
Done